apps/api/src/domain/models/categories/types/CategorizedTree.ts

```diff
@@ -1,6 +1,5 @@
-import { DTO } from '../../../../types/DTO';
 import { Category } from '../entities/category.entity';
 
-export type CategorizedTreeNode = DTO<Category>;
+export type CategorizedTreeNode = Category;
```

Comment:
Is this type really necessary? See if you can do away with it unless you can explain what benefit it gives.